#175756608 Created a Home Page

dianahub250 opened the pull request:
#### What does this PR do?
Creating a home page for Visit Africa App
#### Description of Task to be completed?
- The user can be able to view most visited places
-The use must be able to search 
-The user can be able to navigate to side menu
#### How should this be manually tested?
-Clone it
-Run flutter pub get to install dependencies
-Run it by using flutter run command
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#https://www.pivotaltracker.com/story/show/175756608
#### Screenshots (if appropriate)
![Screenshot_1606125218](https://user-images.githubusercontent.com/71551337/99948814-593e1e00-2d0d-11eb-8b78-b4990eb19044.png)

#### Questions:
N/A

dianahub250 committed:
feature(drawer):add drawer on homepage
have to add drawer to hold user information, notifications and logout
[finishes#175756608]

Sandrine21 reviewed (approved):
Good, you can conyinue.

mujohn26 commented on lib/main.dart:
```diff
@@ -1,117 +1,10 @@
-    );
-  }
+  runApp(MaterialApp(
+    title: 'Todos ',
```
What's the meaning of this tittle ?

mujohn26 commented on test/widget_test.dart:
```diff
@@ -1,30 +1,30 @@
-// utility that Flutter provides. For example, you can send tap and scroll
-// gestures. You can also use WidgetTester to find child widgets in the widget
-// tree, read text, and verify that the values of widget properties are correct.
+// // This is a basic Flutter widget test.
```
What is the use of this file ? I see it's a commented line.

dianahub250 committed:
homepage(add):
Linking images with specific country page and using buttons as widget
[Finishes#175756608]

mujohn26 reviewed (approved):
Good
